Lib/pathlib.py

```diff
@@ -198,14 +213,34 @@ def resolve(self, path, strict=False):
         # Means fallback on absolute
         return None
 
-    def _split_extended_path(self, s, ext_prefix=ext_namespace_prefix):
+    def _split_extended_path(self, s, ext_prefix=ext_namespace_prefix,\
+                             ext_prefix2=ext_namespace_prefix2):
+        # See https://bugs.python.org/issue33898
         prefix = ''
-        if s.startswith(ext_prefix):
+        if s.startswith(ext_prefix) or s.startswith(ext_prefix2):
```

Comment:
Since `ext_prefix` is only used in the `startswith` call, we could simply change the default value to be a tuple of the two prefixes (since `a.startswith(x) or a.startswith(y)` is equivalent to `a.startswith((x, y))`)

Reply:
Done.